ide/coqide/coqide.ml

```diff
@@ -765,6 +765,17 @@ let show_proof_diff where sn =
 
 let show_proof_diffs _ = cb_on_current_term (show_proof_diff `INSERT) ()
 
+let db_cmd cmd sn =  (* todo: still good? *)
+  Coq.try_grab ~db:true sn.coqtop (sn.coqops#process_db_cmd cmd
+    ~next:(function | _ -> Coq.return ()))
+  ignore
+
+let send_db_cmd cmd = cb_on_current_term (db_cmd cmd) ()
+
+(* technique recommended in https://ocaml.org/releases/4.11/htmlman/comp.html#s%3Acomp-errors
```

Comment:
IMO this comment is noise, but it's not utterly necessary to remove it.

Reply:
It was for the benefit of reviewers who strongly objected to a very similar approach in another PR a long while ago.